backup/queries_table_defs.go

```diff
@@ -783,3 +786,24 @@ func selectAsOidToStringMap(connectionPool *dbconn.DBConn, query string) map[uin
 	}
 	return resultMap
 }
+
+type ExtensionTableConfig struct {
+	Oid       uint32
+	Condition string
+}
+
+func GetExtensionTableConfigs(connectionPool *dbconn.DBConn) map[uint32]*string {
+	gplog.Verbose("Retrieving extension table information")
+
+	query := "SELECT unnest(extconfig) oid, unnest(extcondition) condition FROM pg_catalog.pg_extension"
```

Comment:
The query variable seems to be redundant. It is used once. Let's place the string to Select arguments

Reply:
inlined